osu.Game.Rulesets.Osu/OsuRuleset.cs

```diff
@@ -111,7 +111,7 @@ public override IEnumerable<Mod> GetModsFor(ModType type)
                         new OsuModHardRock(),
                         new MultiMod(new OsuModSuddenDeath(), new OsuModPerfect()),
                         new MultiMod(new OsuModDoubleTime(), new OsuModNightcore()),
-                        new OsuModHidden(),
+                        new MultiMod(new OsuModHidden(), new OsuModTraceable()),
```

Comment:
I don't believe this should be a MultiMod with Hidden but rather in the fun section and incompatible to Hidden.

Reply:
Yeah I tried doing that first but since it also inherits from `OsuModHidden`, I can't just put it as incompatible mod or it wouldn't be selectable in game. I couldn't quite figure out how to solve this in `ModSection.cs` so instead I've put it in a multi mod and tweaked the mod tests a tiny bit

Reply:
I'll look into it more tomorrow to make sure I'm not mistaking this mod for something else, but it feels weird that it inherits from Hidden.